eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -35,7 +36,8 @@ steps:
 - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@2
   displayName: CredScan result analysis
   inputs:
-    GdnBreakBaselineFiles: ${{ parameters.BaselineFilePath }}
+    ${{if eq(variables['BaselineExists'], True)}}:
+      GdnBreakBaselineFiles: ${{ parameters.BaselineFilePath }}
```

Comment:
Let's see if we can just pass empty for this if the baseline file isn't needed.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/20749/files